From 6d754ff376cdc27b26fe3e60e7179b7795edf989 Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:32:04 -0400
Subject: [scsi] hpsa: remove messages about volume status VPD inquiry page not supported

Message-id: <1410186734-19986-74-git-send-email-thenzl@redhat.com>
Patchwork-id: 92074
O-Subject: [RHEL6.6 e-stor PATCH 73/83] hpsa: remove messages about volume status VPD inquiry page not supported
Bugzilla: 1069185
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

They are annoying and do not help anyone.

Signed-off-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Reviewed-by: Justin Lindley <justin.lindley@hp.com>
Reviewed-by: Mike Miller <michael.miller@hp.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit 24a4b07879d6d750ef868b94cbdde510d63a1de9)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index 9722de8..4e7ef85 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -2522,27 +2522,21 @@ static int hpsa_get_volume_status(struct ctlr_info *h,
   return HPSA_VPD_LV_STATUS_UNSUPPORTED;
 
  /* Does controller have VPD for logical volume status? */
- if (!hpsa_vpd_page_supported(h, scsi3addr, HPSA_VPD_LV_STATUS)) {
-  dev_warn(&h->pdev->dev, "Logical volume status VPD page is unsupported.\n");
+ if (!hpsa_vpd_page_supported(h, scsi3addr, HPSA_VPD_LV_STATUS))
   goto exit_failed;
- }
 
  /* Get the size of the VPD return buffer */
  rc = hpsa_scsi_do_inquiry(h, scsi3addr, VPD_PAGE | HPSA_VPD_LV_STATUS,
      buf, HPSA_VPD_HEADER_SZ);
- if (rc != 0) {
-  dev_warn(&h->pdev->dev, "Logical volume status VPD inquiry failed.\n");
+ if (rc != 0)
   goto exit_failed;
- }
  size = buf[3];
 
  /* Now get the whole VPD buffer */
  rc = hpsa_scsi_do_inquiry(h, scsi3addr, VPD_PAGE | HPSA_VPD_LV_STATUS,
      buf, size + HPSA_VPD_HEADER_SZ);
- if (rc != 0) {
-  dev_warn(&h->pdev->dev, "Logical volume status VPD inquiry failed.\n");
+ if (rc != 0)
   goto exit_failed;
- }
  status = buf[4]; /* status byte */
 
  kfree(buf);
-- 
1.7.1